.github/workflows/ci.yml

```diff
@@ -0,0 +1,44 @@
+name: 'Continuous Integration'
+on: [ push, workflow_dispatch ]
+jobs:
+  continuousIntegration:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 1.11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.11
+      - name: Login to Docker
+        uses: docker/login-action@v2
+        with:
+          registry: hub.xenit.eu
+          username: ${{ secrets.XENIT_DOCKER_REGISTRY_USERNAME }}
+          password: ${{ secrets.XENIT_DOCKER_REGISTRY_PASSWORD }}
+      - name: Unit tests
+        uses: gradle/gradle-build-action@v2.3.0
+        with:
+          cache-read-only: false
```

Comment:
I'm curious. Why do you set this to false? You want this job to write to cache?

Reply:
We always want cache, not just for master and release, but also for the newly published versions dependencies (e.g. snapshots)